config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: [:show, :index] 
+  resources :posts, only: [:show, :index]
```

Comment:
your statement requires a `do` at the end of this block of code eg 
`
resources :users, only: [:show, :index] do
resources :posts, only: [:show, :index] 
`